src/DemoRunner.php

```diff
@@ -36,7 +36,7 @@ public function runAndPopulateRunResult(RectorRun $rectorRun): void
         $rectorProcessStopwatchEvent = $stopwatch->start('rector-process');
 
         try {
-            $runResult = $this->rectorProcessRunner->run($rectorRun);
+            $runResult = $this->rectorProcessRunner->run($rectorRun->getContent(), $rectorRun->getConfig());
```

Comment:
@TomasVotruba i like this change very much, i was thinking about how to make it not coupled with RectorRun entity, so it could be used event without it and tested easier without mocking - this is possible now.